resources/prometheus/prometheus-rules.yaml

```diff
@@ -291,3 +291,159 @@ spec:
+          record: central:http_incoming_requests:total:rate10m
+
+        # This is a time series of 0s (down) and 1s (up).
+        # Success rate above 65% is floored to 1.
+        # Success rate below 65% is floored to 0.
+        - expr: |
+            floor(
+              (
+                central:grpc_server_handled:OK:rate10m
+              + central:http_incoming_requests:not_5xx:rate10m
+              ) / (
+                central:grpc_server_started:total:rate10m
+              + central:http_incoming_requests:total:rate10m
+              )
+              + 0.35
+            )
+          record: central:sli:error_rate
+
+        # Availability SLI
+        # If any of the SLIs is violated, the service counts as unavailable.
+        # This is a time series of 0s (down) and 1s (up).
+        - expr: |
+            central:sli:pod_ready * on (namespace) group_right() central:sli:error_rate
+          record: central:sli:availability
+
+        - expr: |
+            avg_over_time(central:sli:availability[1h])
+          record: central:sli:availability:avg_over_time1h
+
+        - expr: |
+            avg_over_time(central:sli:availability[28d])
+          record: central:sli:availability:avg_over_time28d
+
+    - name: rhacs-central.slo
+      rules:
+        # Availability SLO
+        - expr: "0.99"
+          record: central:slo:availability
+
+        # 0% exhaustion means no recorded failures.
+        # 100% exhaustion means the SLO target has been reached.
+        # >100% exhaustion means the SLO target has been violated.
+        - expr: |
+            (1 - central:sli:availability:avg_over_time28d) / (1 - scalar(central:slo:availability))
+          record: central:slo:availability:error_budget_exhaustion
+
+        # A burn rate of 1 corresponds to full error budget exhaustion after the SLO window W.
+        # A burn rate of n corresponds to full error budget exhaustion after W/n - in other words,
+        # it measures the exhaustion velocity. To keep the SLO target, a temporary burn rate larger
+        # than 1 must be compensated with a burn rate smaller than 1.
+        - expr: |
+            (1 - central:sli:availability:avg_over_time1h) / (1 - scalar(central:slo:availability))
+          record: central:slo:availability:burnrate1h
+
+    - name: rhacs-central.alerts
+      rules:
+        - alert: Central availability error budget exhaustion - 90%
+          annotations:
+            message: "High availability error budget exhaustion for central. Current exhaustion: {{ $value | humanizePercentage }}."
+            sop_url: "https://gitlab.cee.redhat.com/stackrox/acs-managed-service-runbooks/blob/master/sops/dp-018-rhacs-central-slo-alerts.md"
+          expr: |
+            central:slo:availability:error_budget_exhaustion >= 0.9
+          labels:
+            service: central
+            severity: critical
+            team: appsre
```

Comment:
You can remove the `team` label. It's needed for some process in app-interface, but we don't use it for data plane alerts.